Tools/GenerateWebsiteData.ps1

```diff
@@ -16,8 +16,12 @@ $apiMappings = @{
 
 # Generate a single JSON file containing all the documentation for all the
 # cmdlets. Unfortunately we can't split these into multiple files because of
-# the way we generating web pages in Jekyll.
+# the way we generating web pages in Jekyll/angular..
 $cmdletDocObjects = @()
+
+#Allows us to separate out cmdlets by product.
+$cmdletDocObjectsGcs = @{}
```

Comment:
The higher level hash is used later- see line 93. These are just used to store the cmdlets in their relevant area while it loops through all cmdlets.